guis/desktop/gui.py

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env python3
+################################################################################
+# Project:  ProjPicker (Projection Picker) Desktop GUI
+#           <https://github.com/HuidaeCho/projpicker>
+# Authors:  Owen Smith, Huidae Cho
+#           Institute for Environmental and Spatial Analysis
+#           University of North Georgia
+# Since:    May 27, 2021
+#
+# Copyright (C) 2021 Huidae Cho <https://faculty.ung.edu/hcho/> and
+#                    Owen Smith <https://www.gaderian.io/>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+################################################################################
+import json
+import textwrap
+import pprint
+import wx
+import wx.html2
+from pathlib import Path
+import projpicker as ppik
+from dataclasses import dataclass
+
+
+#################################
+# Constants
+MAP = "openstreet.html"
+
+
+#################################
+# Structs
+@dataclass
+class Geometry:
+    # Struct for easier handling of drawn geometry
+    type: str
+    coors: list or tuple
+
+    def flip(self):
+        """
+        Switch lat lon
+        """
+        corrected_coors = []
+        if self.type == 'Point':
+            corrected_coors = self.coors[1], self.coors[0]
+        else:
+            self.coors = self.coors[0]
```

Comment:
Trying to understand this line. Passing only the first item to `self.coors`?

Reply:
Are you using the left `self.coors` as the pointer to the memory space of the whole `self.coors`? Possible in Python?.. thinking..